source/_docs/ecosystem/caddy.markdown

```diff
@@ -10,12 +10,40 @@ footer: true
 ---
 
 Configure [Caddy Server](https://caddyserver.com/) for use as a reverse proxy to Home Assistant.
+## Install
 
+<p class='note warning'>
+It can potentially be dangerous to send a command straight to bash please make sure you trust the source before running that type of commands, to check what this command does head over to [getcaddy.com](https://getcaddy.com/)
+</p>
+
+Once you are ready to proceed with the installer type:
+```
+curl https://getcaddy.com | bash -s personal
 ```
-hass.example.org {
-    proxy / localhost:8123 {
+
+ 
+Then create a file named ``Caddyfile``:
+```
+hass.yourdomain.org {
```

Comment:
You changed the domain back to example.org on line 44 as suggested by @rohankapoorcom but forgot to do it here.